doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md

```diff
@@ -332,6 +332,7 @@ A configuration example could be:
     cygnus-ngsi.sinks.mongo-sink.mongo_hosts = 192.168.80.34:27017
     cygnus-ngsi.sinks.mongo-sink.mongo_username = myuser
     cygnus-ngsi.sinks.mongo-sink.mongo_password = mypassword
+    cygnus-ngsi.sinks.mongo-sink.mongo_auth_source = admin
     cygnus-ngsi.sinks.mongo-sink.db_prefix = cygnus_
```

Comment:
The new parameter mongo_auth_source should be also added to the configuration table included in this file .md file.